src/limits.rs

```diff
@@ -0,0 +1,70 @@
+//! Run-Time Limits
+
+use libc::c_int;
+
+use ffi;
+pub use ffi::Limit;
+
+use Connection;
+
+impl Connection {
+    /// Returns the current value of a limit.
+    pub fn get_limit(&self, limit: Limit) -> i32 {
```

Comment:
Total nit, but should this be `pub fn limit(...)` instead of `get_limit`? This isn't quite what [the conventions RFC](https://github.com/rust-lang/rfcs/blob/master/text/0344-conventions-galore.md#gettersetter-apis) is talking about with getters/setters (since we take a `Limit` arg) but it's close.